nodejs/week1/avg.js

```diff
@@ -0,0 +1,23 @@
+const averageCalculate = () => {
+    const args = process.argv.slice(2);
+    let sum = 0;
+
+    if (args.length === 0){
+        console.log('ERROR. No arguments');
+        return;
+    }
+
+    for (let j = 0; j < args.length; j++) {
+
+        if(isNaN(args[j])){
+            console.log('ERROR. Some argument is not number');
+            return;
+        }
+        sum += +(args[j]);
```

Comment:
I would consider using parseInt instead of the unary operator at least for better readability.
 [Here](https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which) is an interesting discussion about the differences if you are curious. 
All in all great job, you nailed it Dmytrii :)
And i like that you give your variables meaningfull names and use const and let in the proper way 👍